### ISSUE TITLE
Can't get mouse events to pass through

### ISSUE THREAD

**Issue by dbkaplun** [chjj/ttystudio#14]
Maybe I'm doing something wrong? Just running `ttystudio`, then running my CLI app (slap) in the new shell. Seems like mouse events are just being ignored. iTerm2 2.1.1 on OSX 10.10.4.

**Comment by chjj** [comment]
@dbkaplun, all mouse events, or just "all motion"/"mousemove events"? Tmux 2.0 (if you're using it) no longer supports "all motion" so I had to add a strange workaround for it, but I'm still not sure it works 100% of the time.

What happens when you go to the blessed directory and run: `$ node example/multiplexer.js` and then inside the multiplexer run `$ node test/program-mouse.js`? When you move your mouse or start clicking, do you see any output?

**Comment by chjj** [comment]
Also, make sure you have the latest blessed (that includes the tmux 2.0 workaround if you're using it).

**Comment by dbkaplun** [comment]
Nope, nothing. With just a bare shell inside `ttystudio`, I can't scroll the window at all, but without ttystudio I can scroll fine. (I don't expect the newly scrolled buffer to be picked up by ttystudio as this is handled by the terminal itself from what I understand but this may help debug the issue.)

**Comment by chjj** [comment]
Alright, I'll have to play around with this on iTerm2 right quick.
